src/wallet/wallet_args.cpp

```diff
@@ -140,6 +142,23 @@ namespace wallet_args
 
       auto parser = po::command_line_parser(argc, argv).options(desc_params).positional(positional_options);
       po::store(parser.run(), vm);
+
+      if(command_line::has_arg(vm, arg_config_file))
```

Comment:
Does this block get skipped if you don't give the option ? I remember getting into some trouble with this before. I notice you added a true argument to the declaration, is that the reason (I don't know that code much) ?